jest.config.js

```diff
@@ -0,0 +1,201 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+    // All imported modules in your tests should be mocked automatically
+    // automock: false,
+
+    // Stop running tests after `n` failures
+    // bail: 0,
+
+    // The directory where Jest should store its cached dependency information
+    // cacheDirectory: "/private/var/folders/1l/l1c4p91d1wd3s1t9y8hzqb540000gp/T/jest_dy",
+
+    // Automatically clear mock calls and instances between every test
+    // clearMocks: false,
+
+    // Indicates whether the coverage information should be collected while executing the test
+    collectCoverage: false,
+
+    // An array of glob patterns indicating a set of files for which coverage information should be collected
+    collectCoverageFrom: [
+        './bin/**',
+        './server/**/*.js',
```

Comment:
will this ignore spec files ?

Reply:
@junedkazi What do you mean?
We have our source code (option "collectCoverageFrom") which is covered by tests in .spec.js files (option "testMatch")